# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sjog/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- sjog-0.6/src/sjog_brightness.c.orig	2004-09-17 13:39:19.218635212 +0200
+++ sjog-0.6/src/sjog_brightness.c	2004-09-17 13:39:32.634489573 +0200
@@ -51,8 +51,6 @@
         sjog_set_brightness(currentBrightness);
         break;
 
-     default:
-
    }
 
 }
--- sjog-0.6/src/sjog_scroll.c.orig	2004-09-17 13:42:49.557994523 +0200
+++ sjog-0.6/src/sjog_scroll.c	2004-09-17 13:43:11.730448309 +0200
@@ -63,8 +63,6 @@
         sjog_scroll_up();
         break;
 
-     default:
-
    }
 
 }
